spacewalk/certs-tools/mgr-ssl-cert-setup

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/python3
+#
+# Copyright (c) 2021, SUSE LLC
+#
+# This software is licensed to you under the GNU General Public License,
+# version 2 (GPLv2). There is NO WARRANTY for this software, express or
+# implied, including the implied warranties of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+# along with this software; if not, see
+# http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+#
+# SSL Certificate Setup Tool
+#
+#
+
+
+## language imports
+import sys
+import os
+
+
+mod_name = os.path.basename(sys.argv[0]).replace('-', '_')
+try:
+    mod = __import__("certs." + mod_name)
```

Comment:
Is it required to have such hacky import?

Reply:
Good catch. I will remove it

Reply:
Additionally maybe to run black over `mgr_ssl_cert_setup.py`?